### PR TITLE
Use diffrent name of resources in command-ha tests than in other e2e tests

### PR DESCRIPTION
**Pull request by kaweue**
Currently TestHACommand test may affect TestHAOpenStackServices

### CODE REVIEW

**Comment by kaweue** [comment]
recheck ha

File: test/e2e/ha/command_test.go [hunk 1]
```diff
@@ -338,15 +340,15 @@ func getHACommandCluster(namespace, nodeLabel, storagePath string) *contrail.Man
 
 	webui := &contrail.WebuiService{
 		ObjectMeta: contrail.ObjectMeta{
-			Name:      "webui",
+			Name:      "command-ha",
```
Comment (katrybacka) [inline test/e2e/ha/command_test.go:343]: shouldn't it be command-ha-webui?

File: test/e2e/ha/command_test.go [hunk 2]
```diff
@@ -359,14 +361,14 @@ func getHACommandCluster(namespace, nodeLabel, storagePath string) *contrail.Man
 
 	controls := []*contrail.ControlService{{
 		ObjectMeta: contrail.ObjectMeta{
-			Name:      "control",
+			Name:      "command-ha",
```
Comment (katrybacka) [inline test/e2e/ha/command_test.go:364]: same here with control

File: test/e2e/ha/command_test.go [hunk 3]
```diff
@@ -471,7 +473,7 @@ func getHACommandCluster(namespace, nodeLabel, storagePath string) *contrail.Man
 
 	rabbitmq := &contrail.RabbitmqService{
 		ObjectMeta: contrail.ObjectMeta{
-			Name:      "rabbitmq",
+			Name:      "command-ha",
```
Comment (katrybacka) [inline test/e2e/ha/command_test.go:476]: and here :D

File: test/e2e/ha/command_test.go [hunk 4]
```diff
@@ -526,12 +528,12 @@ func getHACommandCluster(namespace, nodeLabel, storagePath string) *contrail.Man
 		},
 	}}
 	config := &contrail.ConfigService{
-		ObjectMeta: contrail.ObjectMeta{Namespace: namespace, Name: "config", Labels: map[string]string{"contrail_cluster": "command-ha"}},
+		ObjectMeta: contrail.ObjectMeta{Namespace: namespace, Name: "command-ha", Labels: map[string]string{"contrail_cluster": "command-ha"}},
```
Comment (katrybacka) [inline test/e2e/ha/command_test.go:531]: and here

File: test/e2e/ha/command_test.go [hunk 5]
```diff
@@ -553,17 +555,17 @@ func getHACommandCluster(namespace, nodeLabel, storagePath string) *contrail.Man
 	command := &contrail.CommandService{
 		ObjectMeta: contrail.ObjectMeta{
 			Namespace: namespace,
-			Name:      "command",
+			Name:      "command-ha",
```
Comment (katrybacka) [inline test/e2e/ha/command_test.go:558]: aaand...

**Review by katrybacka — CHANGES_REQUESTED** [review]

**Review by kabart — APPROVED** [review]

File: test/e2e/ha/command_test.go (hunk 1 above)
Comment (kaweue) [reply test/e2e/ha/command_test.go:343]: not really, actually I would name all of other resources the same, unfortunately other controllers requires unique names. In my opinion it is rather issue with other controllers that they need to have unique names.

Comment (katrybacka) [reply test/e2e/ha/command_test.go:343]: but it looks really inconsistent right now. I agree and I think we talked about that some time ago that the type in the resource name is unnecessary but right now we have some resources with command-ha names and the others with command-ha-type names :/

Comment (kaweue) [reply test/e2e/ha/command_test.go:343]: Ok I will change that

**Review by katrybacka — APPROVED** [review]

File: test/e2e/ha/command_test.go (hunk 2 above)
Comment (kaweue) [reply test/e2e/ha/command_test.go]: done

File: test/e2e/ha/command_test.go (hunk 3 above)
Comment (kaweue) [reply test/e2e/ha/command_test.go]: fixed

File: test/e2e/ha/command_test.go (hunk 4 above)
Comment (kaweue) [reply test/e2e/ha/command_test.go]: realised

File: test/e2e/ha/command_test.go (hunk 5 above)
Comment (kaweue) [reply test/e2e/ha/command_test.go]: yes

**Comment by kaweue** [comment]
recheck

**Comment by katrybacka** [comment]
recheck

**Comment by kaweue** [comment]
recheck

**Comment by kaweue** [comment]
recheck ha

**Pull request merged**